metricbeat/module/http/_meta/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Tomcat is started to fetch Jolokia metrics from it
```

Comment:
I was thinking of having our own very small web service in go that exposes and http endpoint and sends json to make it clear it is a "dynamic" metricset. Here we have something similar for dropwizard: https://github.com/elastic/beats/tree/master/metricbeat/module/dropwizard/_meta/test Except having a http service will hopefully only be a few lines of code and then in the container `go run ...` can be directly used.